api/download.go

```diff
@@ -9,17 +9,20 @@ import (
 	"path"
 	"strings"
 
+	iclient "github.com/ONSdigital/dp-api-clients-go/v2/identity"
 	dprequest "github.com/ONSdigital/dp-net/v3/request"
 
 	fclient "github.com/ONSdigital/dp-api-clients-go/v2/files"
 	"github.com/ONSdigital/dp-download-service/config"
+	"github.com/ONSdigital/dp-download-service/downloads"
 	"github.com/ONSdigital/dp-download-service/files"
+	filesSDK "github.com/ONSdigital/dp-files-api/files"
```

Comment:
```suggestion
	filesAPIModels "github.com/ONSdigital/dp-files-api/files"
```
I think `filesAPIModels` is a better name for the import as we using the models from it, not the SDK package